docs/03-tutorials/00-eventing/evnt-01-send-legacy-events.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Publish legacy events using Kyma Eventing
+---
+
+Kyma Eventing also supports sending and receiving of legacy events. In this tutorial we will show how to send legacy events using the Eventing Services.
+
+> **NOTE:** It is recommended to use [Cloud Events specification](https://cloudevents.io/) for sending and receiving events in Kyma.
+
+## Prerequisites
+
+1. Provision a [Kyma Cluster](../../02-get-started/01-quick-install.md).
+2. (Optional) Deploy [Kyma Dashboard](../../01-overview/main-areas/ui/ui-01-gui.md) on the Kyma cluster using the following command. Alternatively, you can also use `kubectl` CLI.
+   ```bash
+   kyma dashboard
+   ```
+
+## Create a Function
+
+First, create a sample Function that prints out the received event to console:
+
+<div tabs name="Deploy a Function" group="trigger-workload">
+  <details open>
+  <summary label="Kyma Dashboard">
+  Kyma Dashboard
+  </summary>
+
+1. Go to **Namespaces** and select the default Namespace.
+2. Go to **Workloads** > **Functions** and click **Create Function +**.
+3. Name the Function `lastorder` and click **Create**.
+4. In the inline editor for the Function, replace its source with the following code:
+    ```js
+    module.exports = {
+      main: async function (event, context) {
+        console.log("Received event:", event.data);
+        return;
+      } 
+    }
+    ```
+5. Save your changes.
+6. Wait a few seconds for the Function to have status `RUNNING`.
+
+  </details>
+  <details>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+Run:
+
+```bash
+cat <<EOF | kubectl apply -f -
+  apiVersion: serverless.kyma-project.io/v1alpha1
+  kind: Function
+  metadata:
+    labels:
+      serverless.kyma-project.io/build-resources-preset: local-dev
+      serverless.kyma-project.io/function-resources-preset: S
+      serverless.kyma-project.io/replicas-preset: S
+    name: lastorder
+    namespace: default
+  spec:
+    deps: '{ "dependencies": {}}'
+    maxReplicas: 1
+    minReplicas: 1
+    source: |
+      module.exports = {
+        main: async function (event, context) {
+          console.log("Received event:", event.data);
+          return; 
+        } 
+      }
+EOF
+```
+
+If the resources were created successfully, the command returns this message:
+
+```bash
+function.serverless.kyma-project.io/lastorder created
+```
+
+To check the Function status, run:
+
+```bash
+kubectl get functions -n default lastorder
+```
+
+> **NOTE:** You might need to wait a few seconds for the Function to be ready.
+
+  </details>
+</div>
+
+## Create a Subscription
+
+Next, to subscribe to events, we need a [Subscription](../../05-technical-reference/00-custom-resources/evnt-01-subscription.md) custom resource. We're going to subscribe to events of the type `order.received.v1`.
+All the published events of this type are then forwarded to an HTTP endpoint called `Sink`. You can define this endpoint in the Subscription's spec.
```

Comment:
```suggestion
All the published events of this type are then forwarded to an HTTP endpoint. This endpoint must be configured using the `Sink` attribute in the Subscription's spec. 
```

Reply:
Removed this line, because its already explained in the getting-started guide and also in the subscription CRD doc. It was getting repeated in all the tutorials.